includes/wizards/class-setup-wizard.php

```diff
@@ -520,7 +520,7 @@ public function api_update_theme_with_mods( $request ) {
 				continue;
 			}
 
-			if ( null !== $value && in_array( $key, $this->media_theme_mods ) ) {
+			if ( '' !== $value && in_array( $key, $this->media_theme_mods ) ) {
```

Comment:
if `$value` needs to be an array there, what about a more explicit check like ` ! empty( $value['id'] ) `